angular/src/types/mezon/webview/webview.d.ts

```
@@ -0,0 +1,17 @@
+import { IMezonWebView, MezonEventHandler, MezonAppEvent } from './types';
+export declare class MezonWebView implements IMezonWebView {
+    private eventHandlers;
+    private locationHash;
+    private iFrameStyle;
+    initParams: Record<string, string | null>;
+    isIframe: boolean;
+    constructor();
+    postEvent<T>(eventType: string, eventData: T, callback: Function): void;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)